src/raftstore/store/worker/metrics.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use prometheus::CounterVec;
+
+lazy_static! {
+    pub static ref PD_REQ_COUNTER_VEC: CounterVec =
+        register_counter_vec!(
+            "tikv_raftstore_pd_request_sent_total",
```

Comment:
I don't think so, tikv has a mod named `pd`, I think using `pd`, as a subsystem, is better for that mod.